code/controllers/subsystem/tickets/tickets.dm

```diff
@@ -477,3 +482,13 @@ UI STUFF
 		else
 			message_staff("<span class='admin_channel'>[usr.client] / ([usr]) has taken [ticket_name] number [index]</span>", TRUE)
 		to_chat_safe(returnClient(index), "<span class='[span_class]'>Your [ticket_name] is being handled by [usr.client].</span>")
+
+/datum/controller/subsystem/tickets/proc/unassignTicket(index)
+	var/datum/ticket/T = allTickets[index]
+	if(T.staffAssigned != null)
+		T.staffAssigned = null
+		to_chat_safe(returnClient(index), "<span class='[span_class]'>Your [ticket_name] has been unassigned. Another staff member will help you soon.</span>")
```

Comment:
awkward, they'll know who unassigned because it tells them who takes it

Reply:
doesn't have to. Since everybody can unassign. I've updated the code to ask if you want to unassign others. Makes it more clear